Description: Add support for upstream ChangeLog
 This patch is used to be manually updated in order to
 generate upstream changelog from upstream repository, based
 on current tag or commit diff.
 This file is overriden by mkrelease target from debian/rules
Author: Philippe Thierry
Last-Update: 2017-05-19
--- /dev/null
+++ b/changelog
@@ -0,0 +1,87 @@
+# Release notes for version 67f9137d79e86b808105f1ff061a559900528f5a
+2017-03-28 01:07:12 -0400: [Kevin Amorin] fixes release function
+2017-03-27 23:59:44 -0400: [Kevin Amorin] formating
+2017-03-27 23:58:24 -0400: [Kevin Amorin] Merge pull request #8 from kamorin/fix/readme
+2017-03-27 23:56:28 -0400: [Kevin Amorin] pep8
+2017-03-27 23:41:25 -0400: [Kevin Amorin] adding readme
+2017-03-27 23:19:48 -0400: [Kevin Amorin] comment
+2017-03-27 23:19:12 -0400: [Kevin Amorin] Merge branch 'master' of github.com:kamorin/DHCPig
+2017-03-27 23:18:53 -0400: [Kevin Amorin] comment
+2017-03-27 23:16:08 -0400: [Kevin Amorin] Merge pull request #7 from kamorin/fix/ARP
+2017-03-27 23:10:39 -0400: [Kevin Amorin] fix DHCP response
+2017-03-27 02:08:11 -0400: [Kevin Amorin] remove debugging
+2017-03-27 02:01:46 -0400: [Kevin Amorin] fixed ARP neighbors
+2017-03-27 01:22:18 -0400: [Kevin Amorin] fix dhcp and arp
+2017-03-26 22:04:57 -0400: [Kevin Amorin] Merge pull request #5 from sgeto/master
+2017-03-26 22:03:09 -0400: [Kevin Amorin] Merge pull request #6 from kamorin/fix/fail
+2017-03-26 22:00:23 -0400: [Kevin Amorin] fix fail_cnt comparator
+2017-01-27 17:03:52 +0300: [Ali Abdulkadir] reverted RandMAC() back to randomMAC()
+2017-01-10 04:33:39 +0300: [Ali Abdulkadir] a few subtle enhancements (win32 ready + RandMAC())
+2013-10-06 15:10:12 -0400: [Kevin Amorin] Update pig.py
+2013-10-06 15:05:52 -0400: [Kevin Amorin] update readme, some cleanup
+2013-10-06 15:03:06 -0400: [Kevin Amorin] Update README.md
+2013-04-01 07:25:54 -0700: [Kevin Amorin] Merge pull request #2 from tintinweb/fixups2
+2013-03-29 09:22:56 +0100: [tintin] Update README.md
+2013-03-27 17:27:30 +0100: [tintin] Update pig.py
+2013-03-27 17:24:11 +0100: [tintin] Update README.md
+2013-03-27 15:11:11 +0100: [tintin] update readme
+2013-03-27 14:30:51 +0100: [tintin] + added option to define request option numbers
+2013-03-27 12:45:29 +0100: [tintin] add option to detect dhcp ACKs
+2013-03-26 14:32:20 +0100: [tintin] fixed options
+2013-03-26 12:07:18 +0100: [tintin] beautify :)
+2013-03-25 12:53:10 +0100: [tintin] + threadpool, +minimized-logging, +fix_v6
+2013-03-22 14:15:13 +0100: [tintin] + fixed v6, new options color/client-macs/rc
+2013-01-08 21:44:58 +0100: [tintin] Update README.md
+2012-11-27 15:45:43 +0100: [tintin] fix
+2012-11-27 15:44:17 +0100: [tintin] fix output
+2012-11-27 15:41:43 +0100: [tintin] add v6 neighbor advertisements spoofing
+2012-11-27 13:43:21 +0100: [tintin] v6 action shots
+2012-11-27 13:41:47 +0100: [tintin] actionshots
+2012-11-27 12:59:30 +0100: [tintin] beautify
+2012-11-27 12:57:44 +0100: [tintin] removed comment
+2012-11-27 12:56:55 +0100: [tintin] globals to uppercase vars
+2012-11-27 12:52:48 +0100: [tintin] fixed getop, show options on debug
+2012-11-27 12:35:29 +0100: [tintin] formatting
+2012-11-27 12:34:50 +0100: [tintin] fixed getopt long options
+2012-11-26 22:38:07 +0100: [tintin] Update README.md
+2012-11-26 22:35:35 +0100: [tintin] + more fun
+2012-11-26 22:35:18 +0100: [tintin] + more options, more fun :) switch off harmful attacks and scans by default
+2012-11-26 22:22:49 +0100: [tintin] option -f was missing in getopt
+2012-11-26 22:19:24 +0100: [tintin] Update README.md
+2012-11-26 22:15:17 +0100: [tintin] + i hate global vars! :)
+2012-11-26 22:09:41 +0100: [tintin] minor fixes
+2012-11-26 22:06:32 +0100: [tintin] Update README.md
+2012-11-26 21:46:51 +0100: [tintin] + updated usage
+2012-11-26 21:46:07 +0100: [tintin] update readme
+2012-11-26 21:43:28 +0100: [tintin] Merge branch 'fixups2' of https://github.com/tintinweb/DHCPig into fixups2
+2012-11-26 21:38:24 +0100: [tintin] + added IPv6 Support + added scapy fuzz support
+2012-11-24 00:10:59 +0100: [tintin] Update README.md
+2012-11-24 00:08:35 +0100: [tintin] + beautify __doc__
+2012-11-24 00:06:51 +0100: [tintin] + beautify
+2012-11-24 00:05:43 +0100: [tintin] + more optparser args/options (timeouts and stuff) + slightly changed DoS detection
+2012-11-23 23:46:44 +0100: [tintin] + better help
+2012-11-23 23:35:15 +0100: [tintin] + eyefriendly one-line logging
+2012-11-23 23:28:33 +0100: [tintin] + added show_arp/show_icmp/show_options to getopt (and defaults) + modified filter to also capture arp + added arp handler
+2012-11-23 23:19:27 +0100: [tintin] + fixups - no need to return errorcode if main always returns 0, no need to pass args if main does not take them..
+2012-11-23 23:18:45 +0100: [tintin] + fix indents
+2012-11-23 23:08:35 +0100: [tintin] + harmonized default settings for scapy to use scapy vars
+2012-09-05 15:11:12 -0400: [David LaPorte] Testing2
+2012-09-05 15:07:57 -0400: [David LaPorte] testing
+2012-06-12 00:09:10 -0400: [Kevin Amorin] readme changes
+2012-06-12 00:07:43 -0400: [Kevin Amorin] readme changes
+2012-06-12 00:06:00 -0400: [Kevin Amorin] comment clean up
+2012-06-12 00:05:18 -0400: [Kevin Amorin] readme changes
+2012-06-12 00:01:16 -0400: [Kevin Amorin] readme changes
+2012-05-22 23:55:52 -0400: [Kevin Amorin] .
+2012-05-22 23:51:53 -0400: [Kevin Amorin] .
+2012-05-22 23:51:05 -0400: [Kevin Amorin] v1.21
+2012-05-22 23:50:27 -0400: [Kevin Amorin] v1.2
+2012-05-22 23:48:42 -0400: [Kevin Amorin] v1.1
+2012-05-22 23:47:28 -0400: [Kevin Amorin] v1 readme
+2012-05-22 23:40:34 -0400: [Kevin Amorin] requirements and examples
+2012-05-22 23:17:44 -0400: [Kevin Amorin] updated pig with cli option for interface.  also cleaned up logging
+2012-05-21 23:46:12 -0400: [Kevin Amorin] readme v1
+2012-04-27 06:22:12 -0700: [Kevin Amorin] Merge pull request #1 from dlaporte/master
+2012-04-14 11:34:08 -0400: [Kevin Amorin] adding tigger back in
+2012-03-17 09:53:26 -0400: [David LaPorte] Added interface selection, updated output messages, changed de-fang mechanism
+2012-03-12 23:30:21 -0400: [Kevin Amorin] initial version
\ No newline at end of file
